docs/config_options.md

```diff
@@ -109,7 +109,7 @@ If you define these options you will disable the associated feature, which can s
 * `#define NO_ACTION_ONESHOT`
   * disable one-shot modifiers
 * `#define NO_ACTION_MACRO`
-  * disable all macro handling
+  * disable old style macro handling: MACRO() & action_get_macro
 * `#define NO_ACTION_FUNCTION`
   * disable the action function (deprecated)
```

Comment:
Could you ming adding a reference to `fn_actions` here, for `NO_ACTION_FUNCTION`?

Reply:
ok changed.  I was kind of expecting NO_ACTION_FUNCTION to disable to whole fn_actions array process but not calling action_function is all it seems to do.  Perhaps something for a good first issue.

Reply:
Well, not calling it is functionally the same as disabling it.  Though, code wise... it may not be.